eu.retroshine.RetroShine.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: eu.retroshine.RetroShine
+runtime: org.kde.Platform
+runtime-version: "6.9"
+sdk: org.kde.Sdk
+command: RetroShine
+finish-args:
+  - --share=network
+  - --filesystem=home  # linter error: comment on PR
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --persist=RetroShine
+
+modules:
+  - name: retroshine-binary
+    buildsystem: simple
+    build-commands:
+      - mkdir ${FLATPAK_DEST}/bin
+      # this fails: folder "bin" does not exist!
+      - ln -s ${FLATPAK_DEST}/extra/RetroShine/RetroShine ${FLATPAK_DEST}/bin/RetroShine
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+    sources:
+      - type: extra-data
+        filename: RetroShine-1.1.1-x86_64.tar.gz
+        only-arches:
+          - x86_64
+        # Problem: "url" may cause connection error, "path" is not supported by type "extra-data", only by type "file".
```

Comment:
not sure what you think `path` would do, but I didn't think it does what you think it does.